### PR TITLE
STORM-3028 HdfsSpout: handle empty file in case of ackers

### PR DESCRIPTION
**Pull request by ghajos**
There is a problem with all the storm-hdfs junit tests. The test results may not be valid.

### CODE REVIEW

**Comment by ghajos** [comment]
@srdo Can you please take a look at?

**Comment by revans2** [comment]
@ghajos any plans to address the question from @srdo

**Comment by ghajos** [comment]
@revans2 @srdo I'm sorry! There is no reason to set both commit frequency count and commit frequency sec. I hope that this is already addressed in the last commit.

File: external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/HdfsSpout.java [hunk 1]
```diff
@@ -221,14 +221,18 @@ public void nextTuple() {
...
                     reader = pickNextFile();
                     if (reader == null) {
                         LOG.debug("Currently no new files to process under : " + sourceDirPath);
                         return;
                     } else {
                         fileReadCompletely = false;
+                        newReader = true;
                     }
+                } else {
+                    newReader = false;
```
Comment (srdo) [inline external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/HdfsSpout.java]: Nit: This branch is a no op, newReader is already false

Comment (ghajos) [reply external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/HdfsSpout.java]: Thank you for pointing this out!

**Pull request merged**